workitem_blackbox_test.go

```diff
@@ -606,3 +642,433 @@ func TestPagingDefaultAndMaxSize(t *testing.T) {
 		assert.Fail(t, "Limit is within range", "Expected limit to be %d, got %v", 50, *result.Links.First)
 	}
 }
+
+func getMinimumRequiredUpdatePayload(wi *app.WorkItem) *app.UpdateWorkItemJSONAPIPayload {
+	return &app.UpdateWorkItemJSONAPIPayload{
+		Data: &app.WorkItemDataForUpdate{
+			Type: "workitems",
```

Comment:
now using constants in https://github.com/almighty/almighty-core/pull/482/commits/97411e3fa46127798b490d59166c4713889a36f0.
thanks.